core/src/com/unciv/ui/trade/DiplomacyScreen.kt

```diff
@@ -1008,4 +1012,7 @@ class DiplomacyScreen(
         super.resize(width, height)
         positionCloseButton()
     }
+    val selectedCiv = selectCiv
+    val selectedTrade = selectTrade
```

Comment:
Are these needed?

Reply:
Yes. Without those it can't find them in the function below. I don't really understand why though. It happily accepts viewingCiv but the other 2 it doesn't

Reply:
I made them private now, but if you know how I could get them for the `recreate()` function below, I'll change this